xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -495,6 +507,11 @@
       "track": { "type": "integer" },
       "duration": { "type": "integer" },
       "comment": { "type": "string" },
+	  "contributors": { "$ref": "Audio.Contributors" },
+	  "composer": { "type": "string"},
+	  "conductor": { "type": "string"},
+	  "orchestra": { "type": "string"},
+      "lyricist": { "type": "string"},
```

Comment:
To keep backward compat with non named params, new fields needs to always be added at the end.

+ Some indentation problem